src/components/ng-layer-tune/ng-layer-connector.ts

```diff
@@ -20,10 +21,19 @@ async function getLayer(spec: TGetLayer) {
   const {
     viewerObj,
     layerName,
+    nehubaName
   } = spec
   const viewer = (viewerObj || (window as any).viewer)
+
+  let nehuba  
+
+  await new Promise((rs)=>setTimeout(() => {
+    nehuba = nehubaName && (window as any)[nehubaName]? (window as any)[nehubaName].ngviewer : viewer
+    rs('OK');
+  }, 1000)); 
+
```

Comment:
```suggestion
  const viewer = (viewerObj || (window as any)[ viewerVariableName || 'viewer'])
```

Reply:
for a custom nehuba name, it requires adding `ngviewer` as one more step before access to viewer

Reply:
You can assign .ngviewer to a different variable on the global space

I would not like to closely couple ng layer tune with nehuba